appendices/review-template.md

```diff
@@ -0,0 +1,94 @@
+## Peer Review Template
```

Comment:
@NickleDave @arianesasso @Batalex @cmarmo i have made some changes to this file specifically after looking more closely. if you have a moment in the upcoming weeks please have another look. 

Specifically i was thinking we should call out some items from the packaging guide around code consistency (linting) / PEP 8 and end of life here. In the near future we can then link to packaging guide as it goes online.